.github/workflows/test.yml

```diff
@@ -2,9 +2,9 @@ name: NPM Tests
 
 on:
   push:
-    branches: ["*"]
+    branches: ["**"]
```

Comment:
Oh, yes.

Reply:
Actually, I really hate this "check everything twice" situation where the CI checks both the branch and the PR. What if we check "push" events only for `master` (as we do in Prusti)?

Reply:
It's actually not doing the same twice as `on pullrequest` runs on a simulated commit of merging the pull request into the destination branch

Reply:
I know, but I think that in most cases the simulated merge is enough. I'm considering this only to make economy of macos runners, because I saw them queuing today.

Reply:
Interesting thought. I'm however unaware of an easy mechanism to skip the push trigger if there's an existing PR (to then only execute the PR triggered pipeline)

Reply:
I don't think there is such a mechanism, but it might be fine not to test branches that don't have a PR.